src/server/action_handlers/actionkit-rsvp.js

```diff
@@ -0,0 +1,9 @@
+export const displayName = () => 'ActionKit Event RSVP'
+
+export const available = () => {
+  return true
```

Comment:
Yeah, waiting to see what environment variables are needed in processAction to check here.